docs/response-schema-specified-for-success-status-code.md

```diff
@@ -20,6 +20,69 @@ The ${code} success status code has missing response schema. 200 and 201 success
 
 Validates if 200 & 201 success status codes for an ARM PUT operation has a response schema specified.
 
-## How to fix the violation
+## How to fix 
 
 Ensure that, for 200 & 201 success status codes in a PUT has response schema specified.
+
+## Bad examples
+
+```json
```

Comment:
I noticed this in some other file, maybe it would remove the syntax error highlighting?
```suggestion
```json5
```